src/standalone/Vegetation/Canopy.jl

```diff
@@ -444,36 +444,36 @@ function ClimaLSM.make_update_aux(
         n_leaf = hydraulics.n_leaf
         PlantHydraulics.lai_consistency_check.(n_stem, n_leaf, area_index)
         (; retention_model, conductivity_model, S_s, ν) = hydraulics.parameters
-        @inbounds @. ψ[1] = PlantHydraulics.water_retention_curve(
+        @inbounds @. ψ.:1 = PlantHydraulics.water_retention_curve(
             retention_model,
-            PlantHydraulics.effective_saturation(ν, ϑ_l[1]),
+            PlantHydraulics.effective_saturation(ν, ϑ_l.:1),
             ν,
             S_s,
         )
 
         @inbounds for i in 1:(n_stem + n_leaf - 1)
-
-            @. ψ[i + 1] = PlantHydraulics.water_retention_curve(
+            ip1 = i + 1
```

Comment:
@charleskawczynski I wasnt sure how to avoid defining this intermediate quantity

Reply:
Maybe `.:($$(i+1))` might work? But this looks fine to me, too.

Reply:
It's weird, actually - that works in one case, but not the other. @juliasloan25 
```
julia> @inbounds for i in 1:(n_stem + n_leaf - 1)
             @. ψ.:($$(i+1)) = FT(0)
       end

julia> @inbounds for i in 1:(n_stem + n_leaf - 1)
             @. ψ.:($$(i+1)) = PlantHydraulics.water_retention_curve(
               retention_model,
               PlantHydraulics.effective_saturation(ν, ϑ_l.:($$(i+1))),
               ν,
               S_s,
           )
       end
ERROR: syntax: invalid syntax "ϑ_l.((i + 1))" around REPL[96]:2
Stacktrace:
 [1] top-level scope
   @ REPL[96]:1
```